docs/sources/_index.md

```diff
@@ -2,8 +2,72 @@
 title: Explore Logs documentation
 description: Documentation for Explore Logs
 weight: 100
+hero:
+  title: Grafana Incident
+  level: 1
+  width: 100
+  height: 100
+  description: Grafana Incident is an incident management solution designed to help teams respond quickly and effectively to incidents, outages, and other critical events.
```

Comment:
Do you want to use the name Grafana Incident?

Reply:
Oops - copy and paste error

Reply:
well spotted :)